packages/api/src/payments/payrexxPaymentProvider.ts

```diff
@@ -111,18 +111,46 @@ export class PayrexxPaymentProvider extends BasePaymentProvider {
     }
   }
 
-  async checkIntentStatus(props: CheckIntentProps): Promise<IntentState> {
-    // TODO: fix this
-    /* const intent = await this.stripe.paymentIntents.retrieve(props.payment.intentID)
+  async checkIntentStatus({intentID}: CheckIntentProps): Promise<IntentState> {
+    const signature = crypto.createHmac('sha256', this.instanceAPISecret).digest('base64')
+
+    const res = await fetch(
+      `https://api.payrexx.com/v1.0/Gateway/${intentID}/?instance=${this.instanceName}&ApiSignature=${signature}`,
+      {
+        method: 'GET'
+      }
+    )
+    if (res.status !== 200) throw new Error(`Payrexx response is NOK with status ${res.status}`)
```

Comment:
Could this be
```
`Payrexx response is NOT with status ${res.status}
```

Reply:
Nico, I understood you meant `NOK = NOT OK`, right? Moses you ok with that?

Reply:
yes